app/views/registration_command/_form.erb

```diff
@@ -1,5 +1,5 @@
 <% title _('Register Host') %>
-<%= form_tag({controller: 'registration', action: 'create' }, method: :post, class: 'form-horizontal well') do %>
+<%= form_tag({controller: 'registration_command', action: 'create' }, method: :post, class: 'form-horizontal well') do %>
```

Comment:
```suggestion
<%= form_tag(registration_commands_path, method: :post, class: 'form-horizontal well') do %>
```
Side note: if we would have the RegistrationCommand as a resource, this would be easier.

Reply:
In follow up PR form is going to be replaced with React component anyway